tester/function/testing_call_subroutine.go

```diff
@@ -49,6 +49,8 @@ func Testing_call_subroutine(
 	} else if sub, ok := ctx.Subroutines[name]; ok {
 		state, err = i.ProcessSubroutine(sub, interpreter.DebugPass)
 		i.TestingState = state
+	} else {
+		return value.Null, errors.NewTestingError("subroutine %s not found in VCL", name)
```

Comment:
sounds good. Fixed that. It would be also good to quote the value which makes it easier to spot leading or trailing spaces, however this applies to other messages as well so I did not want to change the style.